tests/unit/components/NcAppSettingsDialog/NcAppSettingsDialog.spec.ts

```diff
@@ -55,8 +55,8 @@ describe('NcAppSettingsDialog: Sections registration', () => {
 
 		await nextTick()
 
-		expect(wrapper.findAll('[role="tab"]')).toHaveLength(1)
-		expect(wrapper.find('[role="tab"]').text()).toBe('test_name')
+		expect(wrapper.findAll('.navigation-list__link')).toHaveLength(1)
```

Comment:
@ShGKme mentioned to me we should rather use semantics than internal class names for testing, so in this case I guess like this?

```suggestion
		expect(wrapper.findAll('nav a')).toHaveLength(1)
```

Reply:
Agree